guides/common/modules/con_applying-custom-configuration.adoc

```diff
@@ -1,37 +1,43 @@
 [id="applying-custom-configuration_{context}"]
 = Applying custom configuration to {ProjectName}
 
-When you install and configure {Project} for the first time by using `{foreman-installer}`, you can specify that the DNS and DHCP configuration files are not to be managed by Puppet by using the installer flags `--foreman-proxy-dns-managed=false` and `--foreman-proxy-dhcp-managed=false`.
-If these flags are not specified during the initial installer run, rerunning of the installer overwrites all manual changes, for example, rerun for upgrade purposes.
-If changes are overwritten, you must run the restore procedure to restore the manual changes.
-For more information, see {InstallingServerDocURL}restoring-manual-changes-overwritten-by-a-puppet-run_{project-context}[Restoring Manual Changes Overwritten by a Puppet Run].
+[IMPORTANT]
+====
+Applying custom configuration to {Project} can have unintended consequences that affect performance or functionality.
+Before applying the custom configuration to your production environment, test your changes in a staging environment or run `{foreman-installer}` with the `--noop` and `--verbose` options.
+If the custom configuration causes problems, remove the offending lines from `/etc/foreman-installer/custom-hiera.yaml` and rerun the `{foreman-installer}`.
+====
 
-To view all installer flags available for custom configuration, run `{installer-scenario} --full-help`.
-Some Puppet classes are not exposed to the {Project} installer.
-To manage them manually and prevent the installer from overwriting their values, specify the configuration values by adding entries to configuration file `/etc/foreman-installer/custom-hiera.yaml`.
-This configuration file is in YAML format, consisting of one entry per line in the format of `<puppet class>::<parameter name>: <value>`.
-Configuration values specified in this file persist across installer reruns.
+The `{installer-scenario} --full-help` command displays all `{foreman-installer}` flags available for custom configuration.
+Additionally, some Puppet classes are not exposed to `{foreman-installer}`.
+You can manage these Puppet classes manually by specifying configuration values in the `/etc/foreman-installer/custom-hiera.yaml` file. 
+Values specified in this file persist when you rerun `{foreman-installer}`.
 
-Common examples include:
+The `/etc/foreman-installer/custom-hiera.yaml` file accepts entries in the following format:
 
-* For Apache, to set the ServerTokens directive to return only the product name:
-+
+[source, yaml, options="nowrap", subs="+quotes,attributes"]
+----
+_Puppet_Module_::__Parameter_Name__: _Parameter_Value_
+----
+
+For a list of available Puppet modules, see the contents of the `/usr/share/foreman-installer/modules/` directory.
+For a list of classes, parameters, and values available for a module, see the `.pp` files in the directory for that module, such as `_Module_Name_/manifests/_Example_PP_File_.pp`.
```

Comment:
We've had a policy (that perhaps wasn't written down explicitly) to avoid documenting this. `custom-hiera.yaml` is an advanced resource we want to discourage users from using. By not documenting it we steer users away from it.

Reply:
I translate that to remove this line, but keep the mention of custom-hiera for upstream.

Reply:
Really? Because I interpreted it in the opposite way: https://github.com/theforeman/foreman-documentation/pull/3752#issuecomment-2806696746  @ekohl Can you please be more specific here? :) Keep the module without line 24, or remove it completely?

Reply:
I think @ehelms is right that we should renove it mostly. On mobile now so I should check it later on my desktop.